actors/migration/nv8/market.go

```diff
@@ -0,0 +1,78 @@
+package nv7
+
+import (
+	"context"
+
+	"github.com/filecoin-project/go-state-types/big"
+	cid "github.com/ipfs/go-cid"
+	cbor "github.com/ipfs/go-ipld-cbor"
+
+	market "github.com/filecoin-project/specs-actors/v2/actors/builtin/market"
+	adt2 "github.com/filecoin-project/specs-actors/v3/actors/util/adt"
+
+	adt3 "github.com/filecoin-project/specs-actors/v3/actors/util/adt"
```

Comment:
Some confused import aliases here. It is tricky to decide what to use, since there is no actual schema change happening. But make sure aliases match the paths.



Alternatively, I think you _could_ just import `/v3` for everything here.

Reply:
I fixed the path for adt2 and changed the alias for market to market2, and introduced a market3 in the hopes of establishing a pattern that will be helpful when schema changes are required.